packages/next/build/index.ts

```diff
@@ -197,6 +197,10 @@ export default async function build(dir: string, conf = null): Promise<void> {
   const pageKeys = Object.keys(mappedPages)
   const dynamicRoutes = pageKeys.filter(page => isDynamicRoute(page))
   const conflictingPublicFiles: string[] = []
+  const errorPageRegex = new RegExp(
+    `^(/|\\\\)_error\\.(?:${config.pageExtensions.join('|')})$`
+  )
+  const hasCustomErrorPage = pagePaths.some(page => page.match(errorPageRegex))
```

Comment:
Can't we just check the value of `createPagesMapping['/_error']`.startsWith('next/')` 🤔 `/_error` will always be there

Reply:
Yeah that works 👍